agenta-web/src/components/Playground/ViewNavigation.tsx

```diff
@@ -34,8 +37,29 @@ const ViewNavigation: React.FC<Props> = ({
         isParamSaveLoading,
         saveOptParams,
         isLoading,
+        isChanged,
     } = useVariant(appName, variant)
 
+    const [unSavedChanges, setUnSavedChanges] = useStateCallback(false)
+
+    useBlockNavigation(unSavedChanges, {
+        title: "Unsaved changes",
+        message:
+            "You have unsaved changes in your test set. Do you want to save these changes before leaving the page?",
```

Comment:
I think you copy pasted the message :) It should say playground and not test set

Reply:
My bad😅